demos/demos_databases_apis/azure_data_explorer/graphistry_ADX_kusto_demo.ipynb

```diff
@@ -0,0 +1,469 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c80d3a57-7c78-4bc3-bb4d-6b8cbd71f9fc",
+   "metadata": {},
+   "source": [
+    "# Persistent Graph Semantics in Azure Data Explorer & Graphistry Integration\n",
+    "\n",
+    "## Microsoft Azure Data Explorer Update\n",
+    "\n",
+    "In a significant step forward for graph analytics, **Microsoft has introduced Persistent Graph Semantics in Azure Data Explorer (ADX)**.  \n",
+    "This feature, announced in [this official Azure update](https://azure.microsoft.com/en-us/updates?id=495985), allows users to define **graph projections as persistent database objects**. \n",
+    "\n",
+    "### Key Benefits:\n",
+    "- Define graph relationships using persisted `graph semantics` that live alongside your ADX tables.\n",
+    "- Simplifies repeatable graph analysis workflows.\n",
+    "- Enables performance optimizations by reusing defined graph topologies.\n",
+    "\n",
+    "This new capability is especially powerful for users working with large-scale telemetry, security, and relational datasets natively in Kusto Query Language (KQL).\n",
+    "\n",
+    "---\n",
+    "\n",
+    "## Graphistry Adopts ADX Persistent Graph Semantics\n",
+    "\n",
+    "The graph visualization and investigation platform **Graphistry** has embraced this enhancement in its latest release.\n",
+    "\n",
+    "### What’s New in Graphistry:\n",
+    "- **Native KQL Support**: You can now use KQL directly within Graphistry to run ADX queries and return dataframes.\n",
+    "- **Persistent Graph Queries**: Pass the name of your persistent graph and perform investigations in Graphistry's GPU-accelerated visual interface..\n",
+    "\n",
+    "\n",
+    "### Why It Matters:\n",
+    "With persistent graph semantics and Graphistry’s GPU-accelerated visual interface, analysts can now:\n",
+    "- Query, explore, and visualize ADX graph data with minimal friction.\n",
+    "- Quickly pivot between raw table views and high-context relationship maps.\n",
+    "- Investigate threats, fraud, IoT, and more—faster and more collaboratively.\n"
```

Comment:
this overall section can be streamlined to maybe half the length

Reply:
If that dry generic stuff is shrunk, then you can expand more here on what the demo is and why it is interesting. This simplifies concrete thinking as they have an example

"Identity data is common everywhere from security & fraud to product & sales. This demo lverage kusto graph to store the data and graphistry to visualize it,s oyou can quickly make insights like patterns, outliers, etc"